solidity/contracts/libraries/operator/Groups.sol

```diff
@@ -409,14 +415,36 @@ library Groups {
     function reportRelayEntryTimeout(
         Storage storage self,
         uint256 groupIndex,
-        uint256 groupSize,
-        uint256 minimumStake
+        uint256 groupSize
     ) public {
+        uint256 punishment = relayEntryTimeoutPunishment(self);
         terminateGroup(self, groupIndex);
         // Reward is limited to min(1, 20 / group_size) of the maximum tattletale reward, see the Yellow Paper for more details.
         uint256 rewardAdjustment = uint256(20 * 100).div(groupSize); // Reward adjustment in percentage
         rewardAdjustment = rewardAdjustment > 100 ? 100:rewardAdjustment; // Reward adjustment can be 100% max
-        self.stakingContract.seize(minimumStake, rewardAdjustment, msg.sender, getGroupMembers(self, groupIndex));
+        self.stakingContract.seize(punishment, rewardAdjustment, msg.sender, getGroupMembers(self, groupIndex));
+    }
+
+    /// @notice Evaluates relay entry timeout punishment using the following
+    /// rules:
+    /// - 1% of the minimum stake for the first 3 months,
```

Comment:
What do you think about giving a bit more precise description about the duration? Month can have 28-31 days. Maybe we can change constants `ONE_MONTH`, `THREE_MONTHS`, `SIX_MONTHS` to say 30 days, 90 days, 180 days so everyone knows exactly?